odl/tomo/backends/astra_setup.py

```diff
@@ -44,7 +43,7 @@
                                   str(astra_ver_num % 100)])
 
     if parse_version(ASTRA_VERSION) < parse_version('1.7'):
```

Comment:
Not sure how to do this meaningfully. If I add `astra_supports('odl')` or so, it's not accurate since *we* decide not to support versions older than some number. I think it's okay to hard-code here since it's the only place. Later on it will still be easy to drop support for more old versions.